### ISSUE TITLE
Language (Hebrew or English)  selection doesn't persist between pages.

### ISSUE THREAD

**Issue by develad** [Maakaf/maakaf-website#284]
### Description

The language (Hebrew or English)  selection doesn't persist between pages. Its work just fine couple commits ago. Something must have been broken in the latest merges/ conflict fixes.

**Comment by Darkmift** [comment]
cannot reproduce.
please provide more info or reproduction steps @develad

**Comment by develad** [comment]
@Darkmift 

https://github.com/Maakaf/maakaf-website/assets/70572668/e6dfbe87-574a-4094-a246-787569711603

**Comment by keren-w** [comment]
I added useTextDirection to ModalContent, this shouldn't have created it but I can try to find out. Do you want to assign me?

**Comment by Darkmift** [comment]
I believe its fixed by:

https://github.com/Maakaf/maakaf-website/blob/b4fc8b49c861eba80a2585f8c0cee85159ac840e/components/Header/LocalSwitcher.tsx#L52

please let me know if not

**Comment by develad** [comment]
@Darkmift 
The bug hasn't resolved

**Comment by Avi-E-Koenig** [comment]
@barel-mishal @Tamir198 
can you reproduce?

**Comment by Tamir198** [comment]
Yes, changing language of the main page is not consistent with the "who are we" page
